docs/shared/monorepo-tags.md

```diff
@@ -289,3 +289,65 @@ We can now restrict projects within the same group to depend on each other based
 ```
 
 There are no limits to number of tags, but as you add more tags the complexity of your dependency constraints rises exponentially. It's always good to draw a diagram and carefully plan the boundaries.
+
+## Banning external imports
+
+**This constraint is only available for projects using ESLint.**
+
+You may want to constrain what external packages a project may import. For example, you may want to prevent backend projects from importing packages related to your frontend framework. You can ban these imports by name by adding a `bannedExternalImports` property to the configuration for a tag.
```

Comment:
I would rephrase `You can ban these imports by name by adding a "bannedExternalImports" property to the configuration for a tag.` to `You can ban these imports using "bannedExternalImports" property in your dependency constraints configuration.`

Reply:
Also, don't forget to update `nx-dev/nx-dev/public/documentation/latest/shared/monorepo-tags.md`